Sources/CvPlayerAI.cpp

```diff
@@ -24495,20 +24478,14 @@ void CvPlayerAI::read(FDataStreamBase* pStream)
 /* BETTER_BTS_AI_MOD                       END                                                  */
 /************************************************************************************************/
 
-	//if (getID() == MAX_PC_PLAYERS)//This MIGHT be unnecessary after making Max PC Players and Max Civ Players different.
-	//{
-	//	//Read NPC data
-	//	int iNum = (MAX_PC_PLAYERS+1);
-	//	for (int iI = iNum; iI < MAX_PLAYERS; iI++)
-	//	{
-	//		PlayerTypes ePlayer = (PlayerTypes)iI;
-	//		//GET_TEAM(eTeam).read2(pStream);
-	//		GET_PLAYER(ePlayer).read2(pStream);
-	//		//TeamTypes eTeam = GET_TEAM(GET_PLAYER(ePlayer).getTeam()).getID();
-	//		//GET_TEAM(eTeam).init(eTeam);
-	//		//GC.getInitCore().setTeam(ePlayer, eTeam);
-	//	}
-	//}
+	if (getID() == MAX_PC_PLAYERS)
+	{
+		//Read NPC data
+		for (int iI = MAX_PC_PLAYERS+1; iI < MAX_PLAYERS; iI++)
+		{
+			GET_PLAYER((PlayerTypes)iI).read(pStream);
```

Comment:
It's just this read that breaks saves right? There's a C2C version that can be used to skip this I think.

Reply:
nevermind. Didn't read before posting.

Reply:
> It's just this read that breaks saves right? There's a C2C version that can be used to skip this I think.

No the read doesn't break saves it is entirely necessary for any NPC units or cities that existed in a save to exist after loading the save.
What makes this branch break old saves is that it changes the enumeration values for all the current NPC teams.